packages/react/src/hooks/useHandleClickOutside.ts

```diff
@@ -0,0 +1,18 @@
+import { RefObject, useEffect } from "react";
+
+export const useHandleClickOutside = (
+  ref: RefObject<HTMLDivElement>,
+  onClose: any
```

Comment:
I think this argument name should be more generic

```suggestion
  onClick: (event: MouseEvent): void
```